packages/synapse-react-client/src/components/SynapseTable/SendToCavaticaConfirmationDialog.tsx

```diff
@@ -102,132 +111,191 @@ export default function SendToCavaticaConfirmationDialog(
   )} to CAVATICA`
 
   return (
-    <ConfirmationDialog
-      open={isShowingExportToCavaticaModal}
-      title="Send to CAVATICA"
-      content={
-        <>
-          <Box
-            sx={{
-              backgroundColor: 'grey.100',
-              border: '1px solid',
-              borderColor: 'grey.300',
-              marginBottom: '15px',
-              padding: '10px 20px 10px 20px',
-            }}
-          >
-            <Typography variant="body1" sx={{ textAlign: 'center' }}>
-              CAVATICA is a data analysis and sharing platform.
-              {cavaticaHelpURL && (
-                <>
-                  {' '}
-                  Read more about CAVATICA{' '}
-                  <Link href={cavaticaHelpURL} target="_blank">
-                    here
-                  </Link>
-                  .
-                </>
-              )}
+    <div>
+      <ConfirmationDialog
+        open={isShowingExportToCavaticaModal}
```

Comment:
This state variable is really confusing me...did you hijack `isShowingExportToCavaticaModal` to now mean showing disclaimer only?
Would it make more sense to have two variables?
```
const showDisclaimer = isShowingExportToCavaticaModal && !disclaimerAcknowledged
const showExport = isShowingExportToCavaticaModal && disclaimerAcknowledged
```

Then pass those into the `open` props?